src/test/java/unicash/logic/commands/ClearTransactionsCommandTest.java

```diff
@@ -32,4 +37,41 @@ public void execute_nonEmptyUniCash_success() {
                 ClearTransactionsCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void multipleClearTransactionsCommand_equalsTrue() {
+        ClearTransactionsCommand clearCommand = new ClearTransactionsCommand();
+        assertEquals(clearCommand, new ClearTransactionsCommand());
+
+    }
+
+    @Test
+    public void sameClearTransactionsCommand_equalsTrue() {
```

Comment:
Both of these have a non-conventional unit test name convention

Reply:
One was a redundant test I missed, thanks I've fixed it